tests/behat/features/asp/engagements/match-engagements-submissions-auth.feature

```diff
@@ -42,11 +30,17 @@ Feature: MATCH Engagements submissions page
     Then I should be on "/node/445/edit"
     When I press "Save"
     Then I should be on "/node/445"
-
-    #TODO: test team column displays mentor, student, consultant, and steering committee assignments
+    #TODO: test team column displays consultant
     # see https://cyberteamportal.atlassian.net/browse/D8-1593
+    When I am on "/match-engagements-submissions"
+    Then I should see "Team"
+    Then I should see "Researcher: Xiaoqin Huang"
+    Then I should see "Mentor: Xiaoqin Huang"
+    Then I should see "Steering Committee: Alana Romanella"
+    Then I should see "Students: Alexandra Lamtyugina"
+
 
-  Scenario: MATCH PM role test match engagements submissions page
+  Scenario: Admin with Match PM role test match engagements submissions page
```

Comment:
This should probably be written as "User with ..."  not "Admin with ..." .  The test is not running as an administrator, so this is a bit misleading.